hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/SCMCommonPlacementPolicy.java

```diff
@@ -127,6 +126,28 @@ public ConfigurationSource getConf() {
    */
   @Override
   public List<DatanodeDetails> chooseDatanodes(
+          List<DatanodeDetails> excludedNodes, List<DatanodeDetails> favoredNodes,
```

Comment:
I think we need to make this method final so it is not mistakenly overridden in a child class.